src/scenarios/shared/sod.py

```diff
@@ -65,6 +65,9 @@ def runtests(self, scenarioname, dirs):
         ]
 
         RunCommand(sod_args, verbose=True).run()
+ 
+        linker_dump_file = os.path.join(APPDIR, 'obj', 'Release', 'netstandard2.1', 'blazor', 'linker', 'linker-dependencies.xml.gz')
```

Comment:
Can we avoid hardcoding `Release` here? If we ever want to run a `Debug` version of this test this will be an issue.

Reply:
It seems that linker dump file will not be generated on Debug.